### ISSUE TITLE
[Tip]Variables with property nodes

### ISSUE THREAD

**Issue by dcapeletti** [mwganson/animator#1]
Hi I have been testing your macro. 
It would seem to me a good handling of the variables, that they have nodes to display their properties. That way it is easier to control 5 properties on 50 or 100 variables.

Regards

**Comment by mwganson** [comment]
You are suggesting Nth, Start, and Step be inside collapsible nodes?

**Comment by dcapeletti** [comment]
Yes, exactly. It's just a matter of view but I think it would help a lot to organize many variables.

**Comment by mwganson** [comment]
The problem is I'm using FreeCAD's properties, so I can't really access the tree widget directly.  Some property types have this collapsible node feature built into them, such as Placement or Vector or Matrix, but not the ones I'm using.
